src/platform/portduino/PortduinoGlue.cpp

```diff
@@ -1,6 +1,7 @@
 #include "CryptoEngine.h"
 #include "PortduinoGPIO.h"
 #include "SPIChip.h"
+#include "RTC.h"
```

Comment:
Trunk is complaining that these are out of alphabetical order. Can you swap RTC and SPIChip?

Reply:
update soon